src/library/scala/runtime/LambdaDeserializer.scala

```diff
@@ -69,7 +71,15 @@ object LambdaDeserializer {
 
       // Lookup the implementation method
       val implMethod: MethodHandle = try {
-        findMember(lookup, getImplMethodKind, implClass, getImplMethodName, implMethodSig)
+        if (targetMethodMap != null) {
```

Comment:
I left it as a convenience for the existing unit test. But I agree its extraneous here and I'll move it into the test itself instead.